src/devices/sonoff.ts

```diff
@@ -32,21 +37,26 @@ const fzLocal = {
     } satisfies Fz.Converter,
 };
 
-const sonoffPrivateCluster = 0xFC11;
+// const sonoffPrivateCluster = 0xFC11;
 const sonoffExtend = {
     addCustomClusterEwelink: () => deviceAddCustomCluster(
         'customClusterEwelink',
         {
             ID: 0xfc11,
             attributes: {
                 radioPower: {ID: 0x0012, type: Zcl.DataType.INT16},
+                radioPowerWithManuCode: {
+                    ID: 0x0012,
+                    type: Zcl.DataType.INT16,
+                    manufacturerCode: Zcl.ManufacturerCode.SHENZHEN_COOLKIT_TECHNOLOGY_CO_LTD,
+                },
                 delayedPowerOnState: {ID: 0x0014, type: Zcl.DataType.BOOLEAN},
                 delayedPowerOnTime: {ID: 0x0015, type: Zcl.DataType.UINT16},
                 externalTriggerMode: {ID: 0x0016, type: Zcl.DataType.UINT8},
                 detachRelayMode: {ID: 0x0017, type: Zcl.DataType.BOOLEAN},
             },
             commands: {
-                protocolData: {ID: 0x01, parameters: [{name: 'data', type: BuffaloZclDataType.LIST_UINT8}]},
+                protocolData: {ID: 0x01, parameters: [{name: 'data', type: 1001}]}, // BuffaloZclDataType.LIST_UINT8
```

Comment:
```suggestion
                protocolData: {ID: 0x01, parameters: [{name: 'data', type: BuffaloZclDataType.LIST_UINT8}]},
```

Could you use this here?

Reply:
https://github.com/Koenkk/zigbee-herdsman-converters/issues/7578
Oh, it looks like it should, but I didn't compile it on this version. If it can be restored in the next version of ZHC, I will update it.